main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: mouth                        
+//  action 1: the mouth speaks         
+mouth.speak();                            
+//  action 2: the mouth spits          
+mouth.spits();                         
+//  action 3: the mout yawns
+mouth.yawn(); 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: ac 
+//  action 1: the ac colds room
+ac.cold();
+//  action 2: the ac warms room
+ac.warm();
+//  action 3: the ac leaks water
+ac.leakWater();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: tree
+//  action 1: the tree produces oxygen
+tree.oxygen();
+//  action 2: the tree does photosynthesis
+tree.photosynthesis();
+//  action 3: the tree grows
+tree.gorw();

```

Comment:
your pseudo-code is missing the verb or it is spelled wrong. 
- `oxygen` is a noun, it is not an action. 
- `produces` is the action word used.
- `does` is the action word used`
- `grow` is spelled wrong`
- [ ] revise